### PR TITLE
Added support for labels to "kubectl stop".

### LINKED ISSUE #5178: "kubectl stop" doesn't support specifying resources by label
The other commands do support it.

```
$ ./cluster/kubectl.sh get pods -l name=frontend
POD                 IP                  CONTAINER(S)        IMAGE(S)            HOST                LABELS              STATUS              CREATED

$ ./cluster/kubectl.sh delete pods -l name=frontend
No resources found

$ ./cluster/kubectl.sh stop pods -l name=frontend
unknown shorthand flag: 'l' in -l
```

### PR DESCRIPTION
**Pull request by jszczepkowski**
Added support for filtering resources by labels and --all to "kubectl stop". Fixes #5178.

### CODE REVIEW

**Comment by piosz** [comment]
It seems it also fixes #4343.

File: pkg/kubectl/cmd/stop.go [hunk 1]
```diff
@@ -46,7 +49,7 @@ func (f *Factory) NewCmdStop(out io.Writer) *cobra.Command {
 		Filenames util.StringList
 	}{}
 	cmd := &cobra.Command{
-		Use:     "stop (<resource> <id>|-f filename)",
+		Use:     "stop (<resource> (<id> | -l <label> | --all) | -f filename)",
```
Comment (bgrant0607) [inline pkg/kubectl/cmd/stop.go]: Please make the usage message consistent with delete:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/delete.go#L63

Comment (jszczepkowski) [inline pkg/kubectl/cmd/stop.go]: Done.

**Comment by jszczepkowski** [comment]
Use string for stop made consistent with delete; use string for delete set straight. PTAL

**Comment by bgrant0607** [comment]
LGTM.

**Pull request merged**